src/database/types.ts

```diff
@@ -134,7 +134,7 @@ export type QueryDraftsArgs = {
   limit?: number
   pagination?: boolean
   sortProperty?: string
-  sortOrder?: string
+  sortOrder?: SortArgs
```

Comment:
We should probably think about handling multiple sorts out of the gate with these DB changes. The way we have these typed will be a breaking change and that is going to be a lot more difficult to handle when it impacts Payload, adapters and user projects.

Reply:
Ah, you mean sorting by two values at the same time instead of just one? What about copying what we did previously? So a simple map:

`sort: { updatedAt: 'desc' },`